src/app/public/modules/vertical-tabset/vertical-tabset.service.ts

```diff
@@ -71,6 +88,25 @@ export class SkyVerticalTabsetService {
     this.tabAdded.next(tab);
   }
 
+  public destroyTab(tab: SkyVerticalTabComponent): void {
+    let tabIndex = this.tabs.indexOf(tab);
+    if (tab.active) {
+      this.destroyContent();
+
+      // Try selecting the next tab first, and if there's no next tab then
+      // try selecting the previous one.
+      let newActiveTab = this.tabs[tabIndex + 1] || this.tabs[tabIndex - 1];
```

Comment:
Is there a risk of getting the `index out of range` error since you could possibly be accessing an index that doesn't exist? For example, try this with a tabset with only one tab?

Reply:
Good question, but I confirmed that trying to access a tab that doesn't exists just returns `undefined`. The `if (newActiveTab)` check below prevents this from throwing any errors.